raiden/api/python.py

```diff
@@ -144,6 +144,24 @@ def leave_token_network(self, token_address):
         connection_manager = self.raiden.connection_manager_for_token(token_address)
         connection_manager.leave()
 
+    def get_connection_manager(self, token_address):
+        """Get the connection manager for a specific token"""
+        connection_manager = self.raiden.connection_manager_for_token(token_address)
+        if not connection_manager:
+            return None
+        else:
+            return connection_manager.funds
```

Comment:
This should be `return connection_manager` or the name of the method should be changed.

Reply:
+1